components/link_tooltip/link_tooltip.tsx

```diff
@@ -19,13 +18,14 @@ const tooltipContainerStyles: CSSProperties = {
 type Props = {
     href: string;
     title: string;
+    dataAttributes: {
```

Comment:
Does it maybe make more sense to make this a global pass-through instead of a few fixed keys? I'm not sure if there's anything value in this, but it would prevent any future issues of this kind.

Reply:
@devinbinnie I originally had it like that but changed it to be more explicit with type safety in mind. Do you have any advice on how to structure the prop definitions to account for extra attributes, maybe an `& any` on the prop? It's not a big deal since `messageToHtmlComponent` is currently Javascript.

But I agree, any further change would only need to applied in one spot if we had it pass all attributes to `LinkTooltip`. Will make the change :+1:

Reply:
I noticed the type issue when editing the test for `LinkTooltip`, which is what brought about the prop change.

Reply:
You can add a dictionary component to the type, so it technically can take in any value.
If TypeScript complains about that, then we can go with `& any` on the type.